.github/workflows/format.yaml

```diff
@@ -0,0 +1,11 @@
+name: Format
+on:
+  push:
+    branches: [ $default-branch ]
```

Comment:
Do we not want to run this for all of our branches?

Also, I haven't tested this so I might be introducing noise, but it is my understanding that `$default-branch` only works in templates, and not in proper workflows. You should use something like `${{ github.event.repository.default_branch  }}` for this to work.

This applies to the other workflow file.

Reply:
I assumed that `$default-branch` worked in all workflows, but you're right it doesn't.

> Do we not want to run this for all of our branches?

The current strategy would be: Run CI in all PRs, and in the main branch. Does that makes sense?

Reply:
Running them in branches is good because you can get feedback without having to open (even a Draft PR). I don't know if there are limits to the number of jobs/minutes we can run though. If there are not, let's default to all branches.

Reply:
CI has been enabled for all `push` events.